docs/rfc/rfc-017-abci++-vote-extension-propag.md

```diff
@@ -553,6 +554,32 @@ Then, we need to handle two new situations, roughly equivalent to cases (h.1) an
   However we can manually have the node crash and recover as a workaround. This effectively converts
   this case into (h.1).
 
+### Upgrade Path
```

Comment:
Is this really a subsection of "Pruning the Extended Commit History" ?
I would either move it up inside the "Base Solution" section, or make it a section at the same level as those ones (IOW, I'd add a "#" at the beginning of this line)

Reply:
It's a subsection of `## Discussion`, using `###` to indicate that it is a subsection of that section.

Reply:
Man... for some reason I was interpreting the number of "#" the other way around 🤦‍♂️